### ISSUE TITLE
Creating custom model for object detection

### ISSUE THREAD

**Issue by Barshan-Mandal** [tensorflow/tensorflow#46846]
How can I create a custom model from a directory containing some images????????????I dont want to download a pre trained model.I want to make my own model for object detection.How can I do that??????

**Comment by Saduf2019** [comment]
@Barshan-Mandal 
Please follow these links: [link](https://keras.io/examples/vision/retinanet/) [link1](https://github.com/jaspereb/Retinanet-Tutorial),[link2](https://d2l.ai/).
AS this is not a bug or feature request, please move this issue to closed status and open a [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow) issue for any further queries.

**Comment by Barshan-Mandal** [comment]
I asked about training using custom images....How can I do that??

**Comment by Barshan-Mandal** [comment]
I dont want to use pre trained model.I asked about training using custom images from a folder....How can I do that??

**Comment by Saduf2019** [comment]
@Barshan-Mandal
A directory containing just images is not enough for object detection, you will have to annotate your images, you can create your own model from scratch the recommended way is to apply transfer learning on pre-trained model that would save a lot of time and computation, yielding good accuracy.
